### DISCUSSION TITLE
What can I do when Nextcloud is not reachable via my domain or if I get `SSL_ERROR_INTERNAL_ERROR_ALERT` or `ERR_SSL_PROTOCOL_ERROR` when opening my Nextcloud domain?

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#2105] [Wiki]
## Problem
In some cases you might get a `SSL_ERROR_INTERNAL_ERROR_ALERT` or `ERR_SSL_PROTOCOL_ERROR` issue when trying to open Nextcloud on your domain.

## Investigate
In that case first, check your Apache container logs via the AIO interface or via `sudo docker logs nextcloud-aio-apache`. 

### Too many certificates already issued for yourdomain.com
If you find in there an issue like `too many certificates already issued for` `yourdomain.com`, you will need to wait until the limit is gone (around a week) or change your domain to e.g. a subdomain: https://github.com/nextcloud/all-in-one#how-to-change-the-domain

### Authorization failed: HTTP 400 urn:ietf:params:acme:error:connection
If you only find log lines like e.g. `authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - [ipv6-address] Connection refused`, you should check if you have a AAAA DNS-record for you domain and remove that for a test and additionally remove any CNAME entry that might be added to your domain for a test. Also make sure that port 443 is correctly forwarded! Additionally, if you should be using cloudflare, make sure to disable their proxy feature as it might block the certificate issuing: https://github.com/nextcloud/all-in-one/discussions/1101 Also check that you are not behind CGNat (you don't have your own public ip-address) and that you are not behind a DS-Lite connection (you only have a public ipv6 but no public ipv4 address).

### Timeout during connect (likely firewall problem)
If you only find log lines about a firewall problem like e.g. `Timeout during connect (likely firewall problem)`, you should check if you have a AAAA DNS-record for you domain and remove that for a test and additionally remove any CNAME entry that might be added to your domain for a test. Also make sure that port 443 is correctly forwarded! Additionally, if you should be using cloudflare, make sure to disable their proxy feature as it might block the certificate issuing: https://github.com/nextcloud/all-in-one/discussions/1101 Also check that you are not behind CGNat (you don't have your own public ip-address) and that you are not behind a DS-Lite connection (you only have a public ipv6 but no public ipv4 address).

---

If things still not work after doing that, feel free to post the nextcloud-aio-apache logs to your thread.

**Comment by AliSabzali** [comment]
> When I click "Open Your Nextcloud" button on containers url page, this page reopen again, and I never can go to login page.
> There is no issue on apache logs.
> it is so weird issue.
> My topology in our network: I have used cloudflare tunnel and pfsene firewall in my network.


![image](https://github.com/nextcloud/all-in-one/assets/10707957/f0bc62cd-8e7f-42b8-b567-d56fe69e5c10)

**apache log file**

![image](https://github.com/nextcloud/all-in-one/assets/10707957/fadc6844-e95d-4374-8da6-49efdadfd8dc)

**Reply by szaimen** [reply]
See https://github.com/nextcloud/all-in-one#how-to-run-nextcloud-behind-a-cloudflare-tunnel

**Reply by norbjert** [reply]
I am having that same issue, whenever I click "open your nextcloud" it just opens a new tab with the same cloud.my.domain/containers URL. I have made sure to disable cloudflares proxy. If I try logging out and logging back in (as described here:https://github.com/nextcloud/all-in-one#how-to-easily-log-in-to-the-aio-interface) I simply get redirected to the cloud.my.domain/login page
![image](https://github.com/nextcloud/all-in-one/assets/89078210/00156bdf-d0f7-4c8c-8ecf-097a7b46205a)

**Reply by szaimen** [reply]
Hi,  can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?

**Reply by norbjert** [reply]
I tried following it to the best of my abilities. I had to add ``--env SKIP_DOMAIN_VALIDATION=true`` as it refused to validate the domain despite everything.
I am unsure what the correct APACHE_IP_BINDING would be, as im neither using localhost nor any IPs, but instead network aliases set in my docker-compose that I reference in my nginx config file. However the issue persists for using 127.0.0.1 and 0.0.0.0 as apache ip binding

**Reply by szaimen** [reply]
If in doubt use `APACHE_IP_BINDING=0.0.0.0` as described. Also did you set e.g. `APACHE_PORT=11000` and  pointed Cloudflare Tunnel at `http://ip.address.of.server.running.aio:11000` ? And indeed `--env SKIP_DOMAIN_VALIDATION=true` is needed for Cloudflare Tunnel.

**Reply by norbjert** [reply]
I managed to get it running, I was using a wrong proxy_pass in my nginx config, at first I was using http://localhost:8080 which only redirected to the AIO setup container again, then I was using http://localhost:11000, after replacing localhost with the actual public IP addr it worked, thanks for your help.

Edit: http://nextcloud.my.domain:11000 also works, which is good since I dont have a static IPv4 addr

**Reply by TheRandumHero** [reply]
So I am having the same issues that @norbjert had. I click the Open Your Nextcloud and it just reloads the same page. I attempted to make the changes you made in Nginx and still no luck. Any other ideas anyone?

My setup is basic. Namecheap with 2 A Records. one with Nextcloud and one with * both pointing to public IP. Nginx with 1 Proxy Host pointing at Nextcloud.mydomain.com. Destination is my internal server IP address at 8081. I have port 8081 set on the Nextcloud Docker too as I wanted to stay away from 8080 if possible.

Would love any advice to try and get this working.

**Reply by szaimen** [reply]
@TheRandumHero see https://github.com/nextcloud/all-in-one/discussions/2105#discussioncomment-6212185

**Reply by TheRandumHero** [reply]
@szaimen forgive my ignorance as I'm not 100% sure where to input the information provided in that reply you tagged. I am currently running all of this on Unraid if that makes a difference. What exactly am I attempting to change?

**Reply by szaimen** [reply]
Can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?

**Comment by housemlm** [comment]
I got both "Authorization failed: HTTP 400 urn:ietf:params:acme:error:connection" and "Timeout during connect (likely firewall problem)". Here is my apache log:
{"level":"error","ts":1686721438.8145874,"logger":"tls.obtain","msg":"will retry","error":"[My Domain] Obtain: [My Domain] solving challenge: My Domain: [My Domain] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - xx.xx.xx.xx: Timeout during connect (likely firewall problem) (ca=https://acme-staging-v02.api.letsencrypt.org/directory)","attempt":10,"retrying_in":3600,"elapsed":7321.574277065,"max_duration":2592000}.
I checked the 80 443 8443 ports,It all listening.Please Help.

**Reply by szaimen** [reply]
Hi, it is written what to do in the article above when you get that log.

**Reply by housemlm** [reply]
> Hi, it is written what to do in the article above when you get that log.

Hi, Thanks for the reply! I've done all the things .But it can't solve the problem. I have a dynamic public IP and using cloudflare. Also 443 is correctly forwarded.

**Reply by szaimen** [reply]
So you removed all AAAA records and CNAME records from your domain for a test?

If so, cloudflare blocks the cert retrieval then.

**Reply by housemlm** [reply]
No AAAA and CNAME records in my domain.Only A records.

**Reply by szaimen** [reply]
> If so, cloudflare possibly blocks the cert retrieval then.

https://github.com/nextcloud/all-in-one/discussions/1101

**Reply by housemlm** [reply]
yeah,i've already read that.i dont use cloudflare proxy. DNS only always .is there any other logs that i can look into?

**Comment by LeoAlecksey** [comment]
Good afternoon. I am using my certificate, how do I add it?

**Comment by amjarman** [comment]
Hi, im getting a strange server error when trying to initialise nextcloud for the first time. 
Also it seems the Apach, Nextcloud & Push are not running but when I look in Portainer they show healthy. 
I have set up NC AIO many time but this is the first time I have done it on a windows PC and it been a nightmare. I have installed Ubuntu on the PC but this has not helped.
I also don't know how to view the error logs from the master container: Tips: To display error details in HTTP response set "displayErrorDetails" to true in the ErrorHandler constructor.
![image](https://github.com/nextcloud/all-in-one/assets/61504173/00f4a865-bd94-49d5-af0d-636c6ff370ff)

![nc](https://github.com/nextcloud/all-in-one/assets/61504173/a52c82a0-5304-4162-b742-b834ffb085f7)

**Reply by szaimen** [reply]
Hi, please open a new thread for this. Thanks!

**Comment by aaroseif** [comment]
Removed AAAA DNS-record  CNAME, added them again for nextcloud.mydomain.
 Ports are forwarded:
![image](https://github.com/nextcloud/all-in-one/assets/74415472/6eaa00f2-a5ac-40e4-bba1-f31ae2e45686)

 logs nextcloud-aio-apache:
 
{"level":"info","ts":1696000122.6663086,"msg":"using provided configuration","config_file":"/tmp/Caddyfile","config_adapter":""}
[Fri Sep 29 15:08:42.668698 2023] [mpm_event:notice] [pid 45:tid 140266037443400] AH00489: Apache/2.4.57 (Unix) configured -- resuming normal operations
[Fri Sep 29 15:08:42.668908 2023] [core:notice] [pid 45:tid 140266037443400] AH00094: Command line: '/usr/local/apache2/bin/httpd -D FOREGROUND'
{"level":"info","ts":1696000122.6698315,"msg":"failed to sufficiently increase receive buffer size (was: 208 kiB, wanted: 2048 kiB, got: 416 kiB). See https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes for details."}
{"level":"info","ts":1696000122.6702414,"msg":"[INFO][FileStorage:/mnt/data/caddy] Lock for 'issue_cert_nextcloud.mydomain' is stale (created: 2023-09-29 15:01:12.349983519 +0000 UTC, last update: 2023-09-29 15:07:37.604784646 +0000 UTC); removing then retrying: /mnt/data/caddy/locks/issue_cert_nextcloud.mydomain.lock"}
{"level":"error","ts":1696000127.3952832,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"nextcloud.mydomain","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"PUBLIC_IPv6: Error getting validation data","instance":"","subproblems":[]}}
{"level":"error","ts":1696000127.395392,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"nextcloud.mydomain","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"PUBLIC_IPv6: Error getting validation data","instance":"","subproblems":[]},"order":"https://acme-v02.api.letsencrypt.org/acme/order/1335216036/211686786596","attempt":1,"max_attempts":3}
{"level":"error","ts":1696000127.3954592,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"nextcloud.mydomain","issuer":"acme-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:connection - PUBLIC_IPv6: Error getting validation data"}
{"level":"error","ts":1696000127.3955266,"logger":"tls.obtain","msg":"will retry","error":"[nextcloud.mydomain] Obtain: [nextcloud.mydomain] solving challenge: nextcloud.mydomain: [nextcloud.mydomain] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - PUBLIC_IPv6: Error getting validation data (ca=https://acme-v02.api.letsencrypt.org/directory)","attempt":1,"retrying_in":60,"elapsed":4.72378731,"max_duration":2592000}
{"level":"error","ts":1696000192.4404438,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"nextcloud.mydomain","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"PUBLIC_IPv6: Error getting validation data","instance":"","subproblems":[]}}
{"level":"error","ts":1696000192.4405138,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"nextcloud.mydomain","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"PUBLIC_IPv6: Error getting validation data","instance":"","subproblems":[]},"order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/120198444/11259318504","attempt":1,"max_attempts":3}

![image](https://github.com/nextcloud/all-in-one/assets/74415472/cd260ca6-6270-4598-a6d7-17df314735ec)

**Comment by S2tX** [comment]
Hello mate. 
Maybe you could help me. I was running nextcloud on my TrueNas Scale just fine until i tried to integrate collabora, so i decided to switch to the AIO after all attempts of setting up my own Collabora Server failed miserably as well as trying to use the CODE App.
Now with AIO I get the SSL_ERROR_INTERNAL_ERROR_ALERT as well. But can't figure out why. I use a clean Ubuntu Server 23.04 LTS. I setup AIO following the guide. I try to set it up for my local domain. After installing to cloud.mydomain.local im hit with the error Message when trying to visit my nextcloud. I tried a couple of things that didn't work. Like a fresh install with the -e SKIP_DOMAIN_VALIDATION=true setting, but that didn't change anything. Right now im on a new fresh install to provide any neccessary logs or whatever. My local dc is running and pointing the ip to cloud.mydomain.local. All other Servers and webservices in my local domain work just fine. https://cloud.mydomain.local:8443/ just gives me what seems to be a timeout. This is what i get from https://cloud.mydomain.local:8080/api/docker/logs?id=nextcloud-aio-mastercontainer.

> Trying to fix docker.sock permissions internally...
> Creating docker group internally with id 995
> ..+......+........+.+..+...+....+.....+.......+...+...+...+..+...+......+..........+.........+.....+.+...+.....+..........+..+....+.....+.......+..............+.+..+......+.+......+...+...........+.+...+......+...+.....+................+......+...+.....+......+.+.....+...+...+..................+...................+..+....+.........+..+......+...+...............+.+......+..+.+...........+++++++++++++++++++++++++++++++++++++++++++++*..+......+.....+......+...+............+......+...+.+........+......+.+.....+....+..+++++++++++++++++++++++++++++++++++++++++++++*........+......+.....+....+.....+.........+......+.+......+...........................+...........+.+...+..................+..+.+......+........+.+...............+......+........+...............+.......+.....+...+...+...+....+..+.+.....+............+.+..+....+...........+...+....+...+...+...........+................+........+.+...........+...............+.......+...+.........+.....+....+..+..........+..+.+..+....+.....+...+......................+.............................+....+.....+.+..............+.............+..+...+.........+.........................+.....+...+...+...+.+...+...........+......+............................+........+...+.+++++
> ...+.....+.+...........+....+...+...+.....+...+.+...+..+++++++++++++++++++++++++++++++++++++++++++++*..+.........+........+.......+...+..+..........+...........+.+++++++++++++++++++++++++++++++++++++++++++++*...............+...+.+......+.................+.+.........+..+..........+...+...+.....+...+........................+...+....+......+..+.+..+.......+.....+......+...................+...+.....+...+....+...+........+............+.........+....+..+....+......+.................+.+............+............+...+........+............+.........+.............+.....+.+..............+...+....+........+......+.........+...+.........+.+..+..........+...+..................+...........+....+......+........+.+............+....................+....+...+...............+........+...+......+......+..................+.......+...+.....+....+............+........+......+................+.....+.+...+............+...+......+...+......+......+........+......+......+.+.................+...+...+.......+........+....+...............+......+......+.........+..............+.......+..............+..........+..+..........+..+...+......................+..+.+..+...+...............+..........+.....+...+.....................+......+....+........+...+...+...............+....+..+....+......+..................+...+.................+.........................+..+..........+...+...+........+....+..+...................+..+....+.........+............+...+...........+.+...+..+...+......+...+...+....+...+......+.................+....+........+...............+.............+..+.+.....+.........+.+.........+...+.....+....+.....+.............+..+..........+.....+...+......+.+...........+.........+...+............................+..+.......+......+........+....+...+.................+.......+......+...+..+....+...+.....+.+............+........+....+............+..................+.................+...+.+..+...............+..........+..................+......+...+..+.........+......+....+............+...+..+.............+.........+......+...........+.+...............+.....+......................+......+..+...+.........+...+............+......+................+.....+.+..............+.+.....+.......+...............+...+...+..+...+.........+.+...............+......+...+......+...........+.........+.+......+.........+...+.....+............................+.................+..........+...+.........+.....+.+..............+...+++++
> -----
> [0;92mInitial startup of Nextcloud All-in-One complete!
> You should be able to open the Nextcloud AIO Interface now on port 8080 of this server!
> E.g. https://internal.ip.of.this.server:8080
> If your server has port 80 and 8443 open and you point a domain to your server, you can get a valid certificate automatically by opening the Nextcloud AIO Interface via:
> https://your-domain-that-points-to-this-server.tld:8443[0m
> {"level":"info","ts":1696361568.6816766,"msg":"using provided configuration","config_file":"/Caddyfile","config_adapter":""}
> [03-Oct-2023 19:32:48] NOTICE: fpm is running, pid 110
> [03-Oct-2023 19:32:48] NOTICE: ready to handle connections
> [Tue Oct 03 19:32:48.684425 2023] [mpm_event:notice] [pid 105:tid 140635901422408] AH00489: Apache/2.4.57 (Unix) OpenSSL/3.1.2 configured -- resuming normal operations
> [Tue Oct 03 19:32:48.684582 2023] [core:notice] [pid 105:tid 140635901422408] AH00094: Command line: 'httpd -D FOREGROUND'
> NOTICE: PHP message: 404 Not Found
> Type: Slim\Exception\HttpNotFoundException
> 4
> Message: Not found.
> File: /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/RoutingMiddleware.php
> Line: 76
> Trace: #0 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/RouteRunner.php(56): Slim\Middleware\RoutingMiddleware->performRouting(Object(GuzzleHttp\Psr7\ServerRequest))
> #1 /var/www/docker-aio/php/vendor/slim/csrf/src/Guard.php(476): Slim\Routing\RouteRunner->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #2 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(168): Slim\Csrf\Guard->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Slim\Routing\RouteRunner))
> #3 /var/www/docker-aio/php/vendor/slim/twig-view/src/TwigMiddleware.php(115): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #4 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Views\TwigMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
> #5 /var/www/docker-aio/php/src/Middleware/AuthMiddleware.php(38): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #6 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(269): AIO\Middleware\AuthMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
> #7 /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/ErrorMiddleware.php(76): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #8 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Middleware\ErrorMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
> #9 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #10 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(199): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #11 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(183): Slim\App->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #12 /var/www/docker-aio/php/public/index.php(182): Slim\App->run()
> #13 {main}
> Tips: To display error details in HTTP response set "displayErrorDetails" to true in the ErrorHandler constructor.
> 

And running sudo docker logs nextcloud-aio-apache get me this:

> Waiting for Nextcloud to start...
> Waiting for Nextcloud to start...
> Waiting for Nextcloud to start...
> Waiting for Nextcloud to start...
> Waiting for Nextcloud to start...
> Waiting for Nextcloud to start...
> Waiting for Nextcloud to start...
> Waiting for Nextcloud to start...
> {"level":"info","ts":1696371044.5447302,"msg":"using provided configuration","config_file":"/tmp/Caddyfile","config_adapter":""}
> {"level":"info","ts":1696371044.564223,"msg":"failed to sufficiently increase receive buffer size (was: 208 kiB, wanted: 2048 kiB, got: 416 kiB). See https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes for details."}
> {"level":"info","ts":1696371044.568286,"msg":"[INFO][FileStorage:/mnt/data/caddy] Lock for 'issue_cert_cloud.mydomain.local' is stale (created: 2023-10-03 21:14:48.686211204 +0000 UTC, last update: 2023-10-03 22:08:34.806654093 +0000 UTC); removing then retrying: /mnt/data/caddy/locks/issue_cert_cloud.mydomain.local.lock"}
> {"level":"error","ts":1696371044.5706887,"logger":"tls.obtain","msg":"will retry","error":"[cloud.mydomain.local] Obtain: subject does not qualify for a public certificate: cloud.mydomain.local","attempt":1,"retrying_in":60,"elapsed":0.001231277,"max_duration":2592000}
> [Tue Oct 03 22:10:44.584171 2023] [mpm_event:notice] [pid 57:tid 140543426796360] AH00489: Apache/2.4.57 (Unix) configured -- resuming normal operations
> [Tue Oct 03 22:10:44.584490 2023] [core:notice] [pid 57:tid 140543426796360] AH00094: Command line: '/usr/local/apache2/bin/httpd -D FOREGROUND'
> {"level":"error","ts":1696371104.5711918,"logger":"tls.obtain","msg":"will retry","error":"[cloud.mydomain.local] Obtain: subject does not qualify for a public certificate: cloud.mydomain.local","attempt":2,"retrying_in":120,"elapsed":60.00173393,"max_duration":2592000}
> {"level":"error","ts":1696371224.5720806,"logger":"tls.obtain","msg":"will retry","error":"[cloud.mydomain.local] Obtain: subject does not qualify for a public certificate: cloud.mydomain.local","attempt":3,"retrying_in":120,"elapsed":180.002622316,"max_duration":2592000}
> {"level":"error","ts":1696371344.573427,"logger":"tls.obtain","msg":"will retry","error":"[cloud.mydomain.local] Obtain: subject does not qualify for a public certificate: cloud.mydomain.local","attempt":4,"retrying_in":300,"elapsed":300.00396876,"max_duration":2592000}
> {"level":"error","ts":1696371644.591616,"logger":"tls.obtain","msg":"will retry","error":"[cloud.mydomain.local] Obtain: subject does not qualify for a public certificate: cloud.mydomain.local","attempt":5,"retrying_in":600,"elapsed":600.022153751,"max_duration":2592000}
> {"level":"error","ts":1696372244.5928197,"logger":"tls.obtain","msg":"will retry","error":"[cloud.mydomain.local] Obtain: subject does not qualify for a public certificate: cloud.mydomain.local","attempt":6,"retrying_in":1200,"elapsed":1200.023361522,"max_duration":2592000}
> {"level":"error","ts":1696373444.5939505,"logger":"tls.obtain","msg":"will retry","error":"[cloud.mydomain.local] Obtain: subject does not qualify for a public certificate: cloud.mydomain.local","attempt":7,"retrying_in":1200,"elapsed":2400.024492096,"max_duration":2592000}
> {"level":"error","ts":1696374644.595263,"logger":"tls.obtain","msg":"will retry","error":"[cloud.mydomain.local] Obtain: subject does not qualify for a public certificate: cloud.mydomain.local","attempt":8,"retrying_in":1800,"elapsed":3600.025804744,"max_duration":2592000}


Thank you very much in advance.

May i add if it was unclear before. The initial Setup via cloud.mydomain.local:8080 works just fine, i can still visist it and all Containers are running. 

Just tried a fresh graphical install of Ubuntu with the same outcome. Could reach the AIO Setup just fine via sub.mydomain.local, localhost or ip. But after all cant connect to the nextcloud anyhow.

Best regards.
